test/rack/session/redis_test.rb

```diff
@@ -309,6 +309,7 @@
   end
 
   # anyone know how to do this better?
+  # FIXME: This test doesn't fail even if multithreaded is set to false!
```

Comment:
if this isn't failing when it should, maybe we should delete it?

Reply:
I'm not sure. It occurred to me later, that it may only fail in JRuby or Rubinius.

Reply:
Nope. I can't get it to fail in JRuby either.

I don't have a strong opinion about whether the test should stay or go, in part because I find the test to be inscrutable, and I can't tell what it's supposed to do.

Reply:
The comment above doesn't really give me much confidence either. I think you should remove it.